backend/dal/Services/Concrete/ParcelService.cs

```diff
@@ -92,6 +92,18 @@ public IEnumerable<Parcel> Get(ParcelFilter filter)
                 query = query.Where(p => agencies.Contains(p.AgencyId));
             }
 
+            if (!String.IsNullOrWhiteSpace(filter.Description))
```

Comment:
We can now search for Description, Municipality, Zoning, ZoningPotential

Reply:
Same comment as above.